scripts/jest/config.source-persistent.js

```diff
@@ -9,6 +9,12 @@ module.exports = Object.assign({}, baseConfig, {
     'ReactIncrementalTriangle',
     'ReactIncrementalReflection',
     'forwardRef',
+    // ReactFreshBabelPlugin is only available for dev.
+    // We need two tests here because otherwise, ReactFreshBabelPlugin-test will
+    // fail due to obsolete snapshots
+    process.env.NODE_ENV === 'development'
+      ? '<rootDir>/packages/react-refresh/src/__tests__/ReactFreshBabelPluginProd-test.js'
+      : '<rootDir>/packages/react-refresh/src/__tests__/ReactFreshBabelPlugin-test.js',
```

Comment:
This is kinda hacky. Can we figure out another way without making exceptions at this level?